File: 백준/Bronze/1076. 저항/저항.py
colors = [
    "black", "brown", "red", "orange", "yellow",
    "green", "blue", "violet", "grey", "white"
]

color1 = input().strip()
color2 = input().strip()
color3 = input().strip()

first = colors.index(color1)
second = colors.index(color2)
multiplier = colors.index(color3)

result = (first * 10 + second) * (10 ** multiplier)
print(result)

File: 백준/Bronze/1076. 저항/저항.cs
namespace ConsoleApp1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            var colorIndex = new Dictionary<string, int>()
            {
                {"black", 0},
                {"brown", 1},
                {"red", 2},
                {"orange", 3},
                {"yellow", 4},
                {"green", 5},
                {"blue", 6},
                {"violet", 7},
                {"grey", 8},
                {"white", 9},
            };

            string color1 = Console.ReadLine();
            string color2 = Console.ReadLine();
            string color3 = Console.ReadLine();

            int num = 0;

            int first = colorIndex[color1];
            int second = colorIndex[color2];
            int multiplier = colorIndex[color3];

            long result = (first * 10 + second) * (long)Math.Pow(10, multiplier);
            Console.WriteLine(result);
        }
    }
}